HandsOnTutorial/1-hello-world.fsx

```diff
@@ -17,47 +21,43 @@ common with MBrace.
 
 A guide to creating a cluster is [here](http://www.m-brace.net/#try).
 
-> NOTE: Before running, build this solution to get the required nuget packages, and edit credentials.fsx to enter your Azure connection strings.
+> NOTE: Before running, build this solution to get the required nuget packages, and edit Azure.fsx to enter your Azure connection strings.
 
 First connect to the cluster using a configuration to bind to your storage and service bus on Azure.
-*)
-let cluster = MBraceAzure.GetHandle(config)
-
-// Optionally, attach console logger to client object 
-cluster.EnableClientConsoleLogger <- true
+*)    
 
 // You can connect to the cluster and get details of the workers in the pool:
 cluster.ShowWorkerInfo()
 
-// You can view the history of processes:
-cluster.ShowProcessInfo()
-
 (** Now execute your first cloud workflow and get a handle to the running job: *)
-let job = 
+let task = 
     cloud { return "Hello world!" } 
-    |> cluster.CreateProcess
+    |> cluster.CreateCloudTask
```

Comment:
Could you create an issue in MBrace.Core for this? 

In short, the renaming was performed with a unification of the old Process<'T> (which lived in the client only and was non-serializable) with the serializable, MBrace.Core CloudTask<'T>, from which nested cloud tasks can be created. Also, we avoided the use the term 'job' since internally this is the name reserved for work items dequeued by machines in the cluster.